src/util/webcrypto.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @flow
+ * @file Wrapper to provide a consistent interface for the webcrypto API
+ * @author Box
+ */
+
+/**
+ * Returns the correct crypto library based on browser implementation
+ * @returns {Object}
+ */
+function getCrypto() {
+    return window.crypto || window.msCrypto || window.webkitCrypto;
```

Comment:
Don't think you need the last one anymore, double check safari.
https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto

Reply:
checked and removed